modules/video-thumbnail.php

```diff
@@ -49,12 +50,19 @@
 			<strong>Access</strong>
 			<?php
 				if ( $public_status == 'Public' ) {
-					if ( $youtube_link ) {
+					if ( $youtube_link && $youtube_link != "No ID") {
```

Comment:
Could we use a Regex to check that `$youtube_link` is a valid url? That way any other invalid values will be caught as well.

Same goes for `$dropbox_link`. Scrolling through the Airtable view, I see a few videos where dropbox_link is "none"

Reply:
Oh that's a good idea. Let's do that! But perhaps we merge+deploy this change now and create the regex as its own ticket?